buildSrc/src/main/java/Dependencies.kt

```diff
@@ -6,6 +6,7 @@ object Sdk {
 
 object Versions {
     const val ANDROIDX_TEST_EXT = "1.1.2"
+    const val ANDROIDX_TEST_EXT_KTX = "1.1.2"
```

Comment:
This extra version number is probably not needed and you can reuse `ANDROIDX_TEST_EXT`

Reply:
I've addressed it.